src/components/app/App.js

```diff
@@ -41,7 +42,10 @@ class App extends React.Component {
           {user.loggedIn &&
             <div>
               <Route path={`${match.url}dashboard`} component={Dashboard} />
-              <Route path={`${match.url}areas`} component={Areas} />
+              <Switch>
+                <Route exact path={`${match.url}areas`} component={Areas} />
+                <Route exact path={`${match.url}areas/create`} component={AreasManage} />
```

Comment:
I think this is required to prevent /areas/create/i-love-react-router rendering the create page. It isn't super important but I think a user would be surprised to find a lovely map at that endpoint instead of a picture of Zuckerberg looking wired. Maybe better practice for 404 handling.